vcpkg.json

```diff
@@ -49,11 +46,31 @@
     },
     {
       "name": "libmicrohttpd",
-      "version-string": "0.9.63#7"
+      "version-string": "0.9.63#8"
+    },
+    {
+      "name": "g3log",
+      "version-string": "1.3.4#4"
     },
     {
       "name": "schnorr",
       "version": "8.2.0"
```

Comment:
Is this correct? Shouldn't it be `8.3.0#1` like for cryptoutils?

Reply:
No, this is correct. The version is actually 1.3.4 and it's our port (4).